Modules/Devices/src/Devices.Infrastructure/PushNotifications/Connectors/Apns/ApnsMessageBuilder.cs

```diff
@@ -71,7 +75,8 @@ public HttpRequestMessage Build()
     private class Payload
     {
         [JsonPropertyName("notId")]
-        public int NotificationId { get; set; }
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+        public string? NotificationId { get; set; }
```

Comment:
as discussed - we don't need this at all.

Reply:
Oops. I forgot about that.